src/unix/os390.c

```diff
@@ -666,6 +666,10 @@ static int os390_message_queue_handler(uv__os390_epoll* ep) {
   __a2e_l(msg.__rfim_utok, sizeof(msg.__rfim_utok));
   handle = *(uv_fs_event_t**)(msg.__rfim_utok);
   assert(handle != NULL);
+
+  if (!uv__is_active(handle))
```

Comment:
```suggestion
  assert(handle->flags & UV_HANDLE_CLOSED == 0);
  if (!uv__is_active(handle))
```

Reply:
Can you move the `if (handle->path != NULL)` later in the function too? There shouldn't be anyway it could be NULL after here, and anyways, it looks like the `uv__basename_r` may have segfaulted if it was?